### PR TITLE
Single visible wall test case

### PR DESCRIPTION
**Pull request by ykulbak**

### CODE REVIEW

**Comment by leosobral** [comment]
Tested Ok in Bimsync

File: Test Cases/v3.0/Visualization/Single visible wall/unzipped/bcf.version [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Version VersionId="3.0">
+  <DetailedVersion>3.0</DetailedVersion>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/Single visible wall/unzipped/bcf.version:3]: Remove detailed version

File: Test Cases/v3.0/Visualization/Single visible wall/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# \<Single invisible wall\>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/Single visible wall/README.md:1]: Should this be 'visible' instead

**Review by pasi-paasiala — CHANGES_REQUESTED** [review]
-1

Imports fine in Solibri, some minor things.

**Comment by ykulbak** [comment]
@pasi-paasiala @jasollien I've applied the review comments please check

**Review by pasi-paasiala — APPROVED** [review]
+1

**Pull request merged**